src/features/query-bar/components/filter/ValueField.js

```diff
@@ -20,6 +20,7 @@ type Props = {
 
```

Comment:
I committed this change with the wrong message, it should be 'refactor value in ValueField'